Microsoft.Recognizers.Text.DateTime.Tests/English/TestTimeParser.cs

```diff
@@ -32,6 +32,7 @@ public void TestTimeParse()
         {
             var today = DateObject.Today;
             int year = today.Year, month = today.Month, day = today.Day, min = 0, second = 0;
+            BasicTest("I'll be back at 7ampm", new DateObject(year, month, day, 7, min, second));
```

Comment:
@tellarin, this is a special processing for Mercury only. They will do some hotfix (add ampm after the number) to distinguish between a selection and a time mention. "at 7ampm" equals to "at 7".